crates/datasources/src/excel/stream.rs

```diff
@@ -0,0 +1,70 @@
+use std::pin::Pin;
+use std::sync::{Arc, Mutex};
+use std::task::{Context, Poll};
+
+use datafusion::arrow::datatypes::{Schema, SchemaRef};
+use datafusion::arrow::record_batch::RecordBatch;
+use datafusion::error::DataFusionError;
+use datafusion::execution::TaskContext;
+use datafusion::physical_plan::streaming::PartitionStream;
+use datafusion::physical_plan::{RecordBatchStream, SendableRecordBatchStream};
+use futures::{Stream, StreamExt};
+
+use crate::excel::errors::ExcelError;
+
+pub type SendableSheetStream = Pin<Box<dyn Stream<Item = Result<RecordBatch, ExcelError>> + Send>>;
+
+pub struct ExcelStream {
+    schema: Arc<Schema>,
+    stream: Pin<Box<dyn Stream<Item = Result<RecordBatch, ExcelError>> + Send>>,
```

Comment:
does the `SendableRecordBatchStream` alias cover this type?

Reply:
I believe it _should_ given:
```rust
impl From<ExcelError> for DataFusionError {
    fn from(e: ExcelError) -> Self {
        DataFusionError::Execution(e.to_string())
    }
}
```

Reply:
it no longer does.